src/components/Collection/Collection.stories.tsx

```diff
@@ -0,0 +1,510 @@
+import React from 'react'
+import { Collection } from './Collection'
+import { CollectionItem } from './CollectionItem'
+import { CollectionHeading } from './CollectionHeading'
+import { CollectionDescription } from './CollectionDescription'
+import { CollectionMeta } from './CollectionMeta'
+import { CollectionMetaItem } from './CollectionMetaItem'
+import { CollectionMetaItemTag } from './CollectionMetaItemTag'
+import { CollectionThumbnail } from './CollectionThumbnail'
+import { CollectionCalendarDate } from './CollectionCalendarDate'
+
+export default {
+  title: 'Components/Collection',
+  component: Collection,
+  parameters: {
+    docs: {
+      description: {
+        component: `
+### USWDS 2.10.0 Collection component
+        `,
+      },
+    },
+  },
+}
+
+export const collectionWithDefaultItems = (): React.ReactElement => (
+  <div className="tablet:grid-col-6">
+    <Collection>
+      <CollectionItem>
+        <CollectionHeading>
+          <a
+            className="usa-link"
+            href="https://www.performance.gov/presidents-winners-press-release/">
+            Gears of Government President’s Award winners
+          </a>
+        </CollectionHeading>
+        <CollectionDescription>
+          Today, the Administration announces the winners of the Gears of
+          Government President’s Award. This program recognizes the
+          contributions of individuals and teams across the federal workforce
+          who make a profound difference in the lives of the American people.
+        </CollectionDescription>
+        <CollectionMeta aria-label="More Information">
+          <CollectionMetaItem>
+            By Sondra Alnsworth and Constance Lu
+          </CollectionMetaItem>
+          <CollectionMetaItem>September 30, 2020</CollectionMetaItem>
+        </CollectionMeta>
+        <CollectionMeta aria-label="Topics">
+          <CollectionMetaItemTag className="usa-tag--new">
```

Comment:
I always look for ways to encapsulate as many of the USWDS classes as possible. What do you think about Suz's suggestion of a `tagStyle` prop where the value `"new"` can apply this class internally?